src/annotator/highlighter.ts

```diff
@@ -339,7 +322,7 @@ export function setHighlightsFocused(highlights, focused) {
       // SVG elements are rendered in document order so to achieve this we need
       // to move the element to be the last child of its parent.
       if (focused) {
-        const parent = /** @type {SVGElement} */ (h.svgHighlight.parentNode);
+        const parent = h.svgHighlight.parentNode as SVGSVGElement;
```

Comment:
The type was changed here from `SVGElement` (ie. _any_ SVG element) to `SVGSVGElement` (ie. specifically the `<svg>` element). Was that intentional? If we don't need to be that specific, I'd prefer to use the more general type.

Reply:
Ah, nice catch.